projects/WeTek_Core/options

```diff
@@ -35,7 +35,7 @@
         TARGET_FEATURES="32bit"
 
         # TARGET_KERNEL_PREFIX
```

Comment:
^^ Change comment too? Or maybe just remove it, as it doesn't add any value - better to describe what the following options does or what it is used for. Same for WeTek_Play.

Reply:
comment has been changed